charts/aws-service-operator/README.md

```diff
@@ -0,0 +1,47 @@
+# AWS Service Operator
+
+This is the Helm chart for the [AWS Service Operator](https://github.com/awslabs/aws-service-operator)
+
+## Prerequisites
+
+- Kubernetes 1.9+
+
+## Installing the chart
+Create AWS resources with Kubernetes
+The chart can be installed by running:
+
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install incubator/aws-service-operator
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the aws-service-operator chart and their default values.
+
+| Parameter                 | Description                            | Default                                            |
+| ------------------------- | -------------------------------------- | -------------------------------------------------- |
+| `image.repository`        | Container image repository             | `awsserviceoperator/aws-service-operator`          |
+| `image.tag`               | Container image tag                    | `v0.0.1-alpha2`                                    |
+| `image.pullPolicy`        | Container pull policy                  | `IfNotPresent`                                     |
+| `operator.accountId       | AWS Account ID to operator on          | `""`                                               |
```

Comment:
Lines 27-35 prefixed with `operator` need a closing tick mark after the key. Check the rendered output to see.